manifests/c/ColonyLabs/ScribeDesktopCapture/4.11.7.0/ColonyLabs.ScribeDesktopCapture.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ColonyLabs.ScribeDesktopCapture
+PackageVersion: 4.11.7.0
+InstallerLocale: en-US
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+UpgradeBehavior: install
+ProductCode: '{bc907fc3-1a05-44f6-be2c-207b9eda0a1d}'
+ReleaseDate: 2024-11-06
+AppsAndFeaturesEntries:
+- DisplayName: Scribe
+  Publisher: Colony Labs, Inc
+  DisplayVersion: 4.11.7.0
```

Comment:
```suggestion
```

DisplayVersion should not be used when it's equal to PackageVersion. This introduced unnecessary client mapping code